app/src/androidTest/java/org/sagebionetworks/research/sageresearch/viewmodel/ScheduleRepositoryTests.kt

```diff
@@ -189,29 +208,15 @@ class ScheduleRepositoryTests: RoomTestHelper() {
             return participantCreatedOn
         }
 
-        override fun cacheSchedules(schedules: List<ScheduledActivityEntity>) {
-            dao.upsert(schedules)
-        }
-
-        override fun updateActivityOnBridge(bridgeSchedule: ScheduledActivity, onNext: Action1<Message>, onError: Action1<Throwable>) {
+        override fun updateSchedulesToBridge(schedules: List<ScheduledActivityEntity>): Completable {
             throwableOnUpdate?.let {
-                onError.call(it)
-            } ?: run {
-                onNext.call(Message())
-            }
-        }
-
-        override fun findSchedule(taskRunUuid: UUID, onNext: Action1<ScheduledActivityEntity>, onError: Action1<Throwable>) {
-            val guid = scheduleTaskRunUuidMap[taskRunUuid] ?: run {
-                onError.call(Throwable("No schedule guid found for taskRunUuid $taskRunUuid, " +
-                        "are you sure you function createScheduleTaskRunUuid() before running the task?"))
-                return
-            }
-            dao.activity(guid).firstOrNull()?.let {
-                onNext.call(it)
+                `when`(activityManager.updateActivities(any()))
```

Comment:
@dephillipsmichael check this out! I have `Mockito` working in an Android test

(Not using Mockito in the best way because I haven't had a chance to refactor, but should reduce the need to create classes like `MockScheduleRepository`)